CMakeLists.txt

```diff
@@ -12,18 +12,16 @@ include(CableToolchains)
 include(CMakePackageConfigHelpers)
 include(GNUInstallDirs)
 
+option(EVMC_TESTING "Build EVMC tests and test tools" OFF)
```

Comment:
Isn't this a foreign key? `${EVMC_TESTING}` instead of `OFF`?

Reply:
No. It's defined before EVMC defines it. I think this line can be dropped actually.

Reply:
I'll just merge this now.

Reply:
I will remove the line.